src/carbonio-files-ui-common/views/components/ListHeader.tsx

```diff
@@ -49,11 +46,7 @@ export interface ListHeaderProps {
 const MAX_ACTIONS_VISIBLE = 3;
 
 export const ListHeader = ({
-	isSelectionModeActive,
-	unSelectAll,
-	selectAll,
 	isAllSelected,
```

Comment:
Could isAllSelected and selectedCount be calculated and included inside the selection hooks?